### PR TITLE
Refactor tests

### PR DESCRIPTION
**Pull request by jukkah**
Grouped tests under the same describes. See changes to output in Travis.

Note that most of changes are space changes (decreased indentation). I isolated them to the second commit while the first commit is only one worth to review.

### CODE REVIEW

**Comment by sdepold** [comment]
There is one failing test now

**Comment by jukkah** [comment]
@sdepold All tests passing.

File: test/test.js [hunk 1]
```diff
@@ -0,0 +1,4 @@
+'use strict';
```
Comment (sdepold) [inline test/test.js]: What is this file for?

Comment (jukkah) [inline test/test.js]: It is main test file but it is not required.

Comment (sdepold) [inline test/test.js]: Since we have the gulp task it seems redundant to me :) Can we remove it?

Comment (jukkah) [inline test/test.js]: Done.

**Pull request merged**